### ISSUE TITLE
Suggested rewording to normative text of Visible Controls

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1932]
A hard SC to parse, especially the preamble:
>Where receiving pointer hover or keyboard focus triggers user interface components to be visible, information needed to identify that user interface components are available is visible, except when:

Always have to reread just to sort out where the verb is. Wondering if it would be better with "Where UICs become visible as a result of pointer hover or keyboard focus, information needed to identify…"

Would become:
> Where UICs become visible as a result of pointer hover or keyboard focus, information needed to identify that user interface components are available is visible, except when:

**Comment by alastc** [comment]
When you spell out UIC I'm not sure it makes much difference?
> Where user interface components become visible as a result of pointer hover or keyboard focus, information needed to identify that user interface components are available is visible, except when:

The 'trippy' bit is: "that user interface components **are** available **is** visible", which both have.

**Comment by mbgower** [comment]
Part of it for me was the use of "triggers" as a verb when it's easy to misread it as a noun at first glance, then you have to go back and mentally turn it into a verb.

That's what I was trying to solve.

**Comment by mbgower** [comment]
Where user interface components become actionable as a result of pointer hover or keyboard focus, the components have pre-existing indicators of the components  such interaction, except when:

**Comment by mraccess77** [comment]
@mbgower it might not just be the user interface components but another component or additional component may become actionable or even present

**Comment by alastc** [comment]
Ok, so a quick PR is needed for this and we can survey it:

Where user interface components become visible as a result of pointer hover or keyboard focus, information needed to identify that user interface components are available is visible, except when:

**Comment by DavidMacDonald** [comment]
I think that we are getting more confusing with the proposed wording.
Perhaps we can address @mbgower 's concerns that "triggers" can be read as a noun, with amore conservative amendment. By switching it to "causes". Also I notice that the language of the preamble quoted by Michael is not the same as the [most recent draft ](https://w3c.github.io/wcag/guidelines/22/#content-on-hover-or-focus)(unless its been amended somewhere else). Here is my suggestion:

> Where receiving and then removing pointer hover or keyboard focus causes additional content to become visible and then hidden, the following are true:

If for some reason the group wanted to remove "and then removing ..." then it could be like this.

> Where receiving pointer hover or keyboard focus causes additional content to become visible, the following are true:

**Comment by DavidMacDonald** [comment]
An attempt to fix 
> "... are available is visible ..."
How about

> Where user interface components become visible as a result of pointer hover or keyboard focus, information needed to identify that user interface components are available is provided visually, except when:

OR this adjustment to use "find" instead of "identify" which seems awkward

> Where user interface component becomes visible as a result of pointer hover or keyboard focus, information needed to perceive that it is available is provided visually, except when:

**Comment by alastc** [comment]
We've tried to combine aspects of other threads into #2019, from talking to a couple of people that seems to be generally clearer and easier to understand. Does anyone on this thread have strong feelings about aspects here that should be included?

**Comment by alastc** [comment]
Closing as we agreed the update to the SC text here: https://www.w3.org/2021/12/03-ag-minutes.html#t16
